Dockerfile

```diff
@@ -1,57 +1,5 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
```

Comment:
@ruiztulio well, as a lot of the refactoring i made depends on ubuntu being in that version, i suggest we hold the merge of this PR until the image is fully tested with our working environment.

Older versions of ubuntu don't have some of the packaged versions of python packages i removed, so i would have to put back pip and the development packages. Also would have to put back postgres version from upstream.